src/sentry/search/events/datasets/metrics.py

```diff
@@ -1853,9 +1854,38 @@ def _resolve_total_performance_score_function(
             for vital in vitals
         }
 
+        weights = {
+            vital: Function(
+                "if",
+                [
+                    Function(
+                        "isZeroOrNull",
```

Comment:
Both a 0 or a null value represent a missing vital score?

This makes sense to me, because you can't ever have an instantaneous load time for any vital metric.

Reply:
The `isZeroOrNull` condition is ran on the `count` of the metric so we're really just checking for the existence of the vital, not the actual value. This is false if an `lcp` with value of `0ms` is reported, because the count is > 0 in this case.

You're also right, it shouldn't really be possible for load time vitals to be 0ms. It might be possible for cls to be 0 though, and that would depend on if the sdk reports it.

Not sure if there's an easier clickhouse or snuba function to do this check, but this works. We'll see if there are any suggestions